tests/testdata/codegen/plain-object-defaults/docs/foo/_index.md

```diff
@@ -360,9 +360,9 @@ foo_resource = example.Foo("fooResource",
         "thinker": "string",
         "answer": 0,
         "plain_other": {
-            "required_arg": "string",
+            "requiredArg": "string",
```

Comment:
Is this correct?

Looking at the python code for this, we have `plain_other: NotRequired['HelmReleaseSettingsArgsDict']`, so I think this should still be using snake_case.

Reply:
Yeh this is wrong, maybe because it's a nested type. I'll add a nested ref object and get back to this.